crates/stellar/README.md

```diff
@@ -0,0 +1,46 @@
+# Bitcoin library
```

Comment:
if we are renaming it, maybe it is also a good idea to rename it in the readme and all around